templates/interface/add-markers.interface.hbs

```diff
@@ -187,7 +187,7 @@
                     <span class="panel-title-text">\{{marker.name}}</span>
                     <span class="fa panel-chevron fa-chevron-up"></span>
                   </h4>
-                  <a href="#" @click.prevent="deleteMarkerStyle(idx)"><span class="icon-delete fa fa-trash"></span></a>
+                  <a href="#" @click.prevent="deleteMarkerStyle(idx, marker)"><span class="icon-delete fa fa-trash"></span></a>
```

Comment:
Update to: `deleteMarkerStyle({ index: idx, marker: marker })`

Reply:
fixed